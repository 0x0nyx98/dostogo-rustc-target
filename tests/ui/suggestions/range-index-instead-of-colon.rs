// edition:2021

fn main() {
    &[1, 2, 3][1:2];
    //~^ ERROR: expected one of
    //~| HELP: you might have meant to make a slice with range index
    //~| HELP: maybe write a path separator here
}