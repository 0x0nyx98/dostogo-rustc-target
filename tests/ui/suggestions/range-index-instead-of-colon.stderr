error: expected one of `.`, `?`, `]`, or an operator, found `:`
  --> $DIR/range-index-instead-of-colon.rs:4:17
   |
LL |     &[1, 2, 3][1:2];
   |                 ^ expected one of `.`, `?`, `]`, or an operator
   |
help: you might have meant to make a slice with range index
   |
LL |     &[1, 2, 3][1..2];
   |                 ~~

error: aborting due to 1 previous error

